cfgov/unprocessed/js/organisms/EmailSignup.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+// Required modules.
+var atomicHelpers = require( '../modules/util/atomic-helpers' );
+var ERROR_MESSAGES = require( '../config/error-messages-config' );
+var FORM_MESSAGES = ERROR_MESSAGES.FORM.SUBMISSION;
+var Notification = require( '../molecules/Notification' );
+
+/**
+ * EmailSignup
+ * @class
+ *
+ * @classdesc Initializes the organism.
+ *
+ * @param {HTMLNode} element
+ *   The DOM element within which to search for the organism.
+ * @returns {EmailSignup} An instance.
+ */
+function EmailSignup( element ) {
+  var BASE_CLASS = 'o-email-signup';
+  var UNDEFINED;
+  var _baseElement = atomicHelpers.checkDom( element, BASE_CLASS );
+  var _formElement = _baseElement.querySelector( 'form' );
+  var _emailElement = _formElement.querySelector( 'input[type="email"]' );
+  var _codeElement = _formElement.querySelector( 'input[name="code"]' );
```

Comment:
If you wanted to shorten these, elsewhere we're using `Dom` in place of `Element`.